src/pages/publication/contributors_tab/components/SortableTable.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { FC } from 'react';
+import { SortableContainer, SortableElement } from 'react-sortable-hoc';
+
+import {
+  Checkbox,
+  FormControlLabel,
+  Table,
+  TableBody,
+  TableCell,
+  TableRow,
+  TextField,
+  Button,
+} from '@material-ui/core';
+import { Contributor } from '../../../../types/contributor.types';
+import { FormikProps, useFormikContext, Field } from 'formik';
+import { Publication } from '../../../../types/publication.types';
+import AddContributor from '../AddContributor';
+import { useTranslation } from 'react-i18next';
+import DeleteIcon from '@material-ui/icons/Delete';
+
+import FormCardSubHeading from '../../../../components/FormCard/FormCardSubHeading';
+import { removeDuplicatesByScn, arrayMove } from '../../../../utils/helpers';
+
+interface SortableItemProps {
+  contributor: Contributor;
+  placement: number;
+  onDelete: (index: number) => void;
+}
+
+const SortableItem = SortableElement(({ contributor, placement, onDelete }: SortableItemProps) => {
+  const { t } = useTranslation();
+
+  const index = placement - 1;
+
+  return (
+    <TableRow tabIndex={0} key={contributor.systemControlNumber}>
+      <TableCell align="left">
+        <FormCardSubHeading>{contributor.name}</FormCardSubHeading>
+        <Field name={`contributors[${index}].corresponding`}>
```

Comment:
Vet ikke hva som er best practise her, men det føles litt rart å legge mange fields inni en field... Kunne dette vært løst med noen props fra `FieldArray` eller `useFormikContext`?

Reply:
kom borti no lignende for uploaded files (FileCard.tsx), hvor jeg sendte ned remove og replace som prop